adam/experiment/observer.py

```diff
@@ -751,6 +754,50 @@ def report(self) -> None:
             self.precision_recall_observer.report()
 
 
+@attrs(slots=True)
+class YAMLLogger(DescriptionObserver[SituationT, LinguisticDescriptionT, PerceptionT]):
+    name: str = attrib(validator=instance_of(str))
+    experiment_path: Path = attrib(validator=instance_of(Path))
+    counter: int = attrib(kw_only=True, default=0)
+
+    def observe(  # pylint:disable=unused-argument
+        self,
+        situation: Optional[SituationT],
+        true_description: LinguisticDescription,
+        perceptual_representation: PerceptualRepresentation[PerceptionT],
+        predicted_descriptions: Mapping[LinguisticDescription, float],
+    ) -> None:
+        output_dict: MutableMapping[str, Any] = dict()
+        output_dict["gold_language"] = true_description.as_token_string()
+        output_dict["output_language"] = []
+
```

Comment:
Is it worth defining strings for `output_language` and `gold_language` that are reused?

Reply:
Yes this makes sense.